pyuvdata/parameter.py

```diff
@@ -312,7 +313,7 @@ class LocationParameter(UVParameter):
     def __init__(self, name, required=True, value=None, spoof_val=None, description='',
                  acceptable_range=(6.35e6, 6.39e6), tols=1e-3):
         super(LocationParameter, self).__init__(name, required=required, value=value,
-                                                spoof_val=spoof_val, form=(3,),
+                                                spoof_val=spoof_val, form=3,
```

Comment:
Why does `form` go from a tuple to a flat value? Is this just to make it so the user doesn't have to specify things as a tuple? (Happens a few other places, this is just the first one I saw.)

Reply:
So we should get @adampbeardsley to weigh in here. My read of the code is that it was always supposed to support a single value as well as a tuple, but there was no testing of the lines that used the single value, so I changed this to a single value to test those lines of code.

Reply:
Actually, I should have said that I added testing for that case. I changed it here because it seemed simpler. (Sorry, got confused about what file we were talking about)

Reply:
I can take it out if we want. It's not super documented and if we want tuples I'm ok with that, I was just trying to cover the code that existed.